### PR TITLE
dockutil 3.0.2

### PR DESCRIPTION
**Pull request by mofneko**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/dockutil.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Dockutil < Formula
   end
 
   bottle do
-    sha256 cellar: :any_skip_relocation, all: "f5f87d9e286c2b294bb157ac9f87baa2720fff044c7a92c0b80b9cb82db8a87e"
+    sha256 cellar: :any_skip_relocation, all: "6dbbc1467caaab977bf4c9f2d106ceadfedd954b6a4848c54c925aff81159a65"
```
Comment (carlocab) [inline Formula/dockutil.rb:14]: Please don't touch the bottle block.

**Comment by p-linnane** [comment]
There have been attempts to update this multiple times: https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+is%3Aclosed+dockutil+3.0.2

There is no activity from upstream in over a year, so this is unlikely to work.

**Comment by mofneko** [comment]
At least it works in the Ventura environment.
Now that the formula exists, doesn't it make sense to update it?

File: Formula/dockutil.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Dockutil < Formula
   desc "Tool for managing dock items"
   homepage "https://github.com/kcrawford/dockutil"
-  url "https://github.com/kcrawford/dockutil/archive/2.0.5.tar.gz"
+  url "https://github.com/kcrawford/dockutil/archive/3.0.2.tar.gz"
   sha256 "6dbbc1467caaab977bf4c9f2d106ceadfedd954b6a4848c54c925aff81159a65"
```
Comment (carlocab) [inline Formula/dockutil.rb:5]: This checksum is wrong.
```suggestion
  sha256 "8d0117fccd836a14782107bdd3619df17dd4708470e3522f5b2ab0456769ba79"
```

**Review by carlocab — CHANGES_REQUESTED** [review]
CI needs to pass before merging.

File: Formula/dockutil.rb (hunk 2 above)
Comment (mofneko) [reply Formula/dockutil.rb]: thanks that's fixed

**Comment by carlocab** [comment]
Please test your install locally first before pushing more changes that may or may not work, and then requesting reviews immediately after doing so.

My [review from above](https://github.com/Homebrew/homebrew-core/pull/132838#pullrequestreview-1461793124) has not changed, so I'm not sure why you're requesting another review. If you have a specific question, then add it as a comment.

**Pull request closed without merging**